### PR TITLE
HackStudio: Remove `release_value_but_fixme_should_propogate_errors`s

### PR DESCRIPTION
**Pull request by ominusliticus**
There were a total of 20 fixmes that were removed. This required me to create a `initialize_all()` function for the HackStudioWidget class that could actually propagate the errors forward to the Serenity::Main function for the HackStudio application.

All the fixmes dealt with loading icons for the various actions possible.
This should not be a failure that keeps HackStudio from running, but currently, if the icons cannot be loaded HackStudio fails to open.

I am currently not familiar enough with serenity programming patterns to know how to handle these errors such that the application doesn't just fail to run. 
Hoping that the comments on the PR will help.

### CODE REVIEW

File: Userland/DevTools/HackStudio/HackStudioWidget.cpp [hunk 1]
```diff
@@ -461,33 +465,58 @@ void HackStudioWidget::set_edit_mode(EditMode mode)
...
+ErrorOr<NonnullRefPtr<GUI::Menu>> HackStudioWidget::create_project_tree_view_context_menu()
 {
-    m_new_file_actions.append(create_new_file_action("&C++ Source File", "/res/icons/16x16/filetype-cplusplus.png", "cpp"));
-    m_new_file_actions.append(create_new_file_action("C++ &Header File", "/res/icons/16x16/filetype-header.png", "h"));
-    m_new_file_actions.append(create_new_file_action("&GML File", "/res/icons/16x16/filetype-gml.png", "gml"));
-    m_new_file_actions.append(create_new_file_action("P&ython Source File", "/res/icons/16x16/filetype-python.png", "py"));
-    m_new_file_actions.append(create_new_file_action("Ja&va Source File", "/res/icons/16x16/filetype-java.png", "java"));
-    m_new_file_actions.append(create_new_file_action("C Source File", "/res/icons/16x16/filetype-c.png", "c"));
+    auto action = TRY(create_new_file_action("&C++ Source File", "/res/icons/16x16/filetype-cplusplus.png", "cpp"));
+    m_new_file_actions.append(action);
```
Comment (mustafaquraish) [inline Userland/DevTools/HackStudio/HackStudioWidget.cpp:471]: if we're handling errors properly anyway, lets do this for appending as well:
```cpp
TRY(m_new_file_actions.try_append(action));
```
(and similar for all the other appends below)

Comment (ominusliticus) [reply Userland/DevTools/HackStudio/HackStudioWidget.cpp]: I was uncertain if I could return from within a function call, as my tests with compiler explorer showed that I couldn't.
But I tries in the code base and it seemed to compile just fine. Cannot say that I know how it works though

**Comment by mustafaquraish** [comment]
Ideally all of this should be one commit, instead of 3. We generally don't like a separate commit fixing something in a previous commit for the same PR. Could you please squash all the commits here? :)

**Comment by ominusliticus** [comment]
I think that squashed them down to one. At least, that's what it looks like on my fork.
Please let me know if I should do anything else! =)

**Comment by AtkinsSJ** [comment]
This has merge conflicts, so you'll need to rebase on the current master branch and fix them up, then push again.

**Review by alimpfard — APPROVED** [review]

**Pull request merged**